_posts/2018-10-04-cockpit-179.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 179
+author: Gundersanne
+date: 2018-10-04
+tags: cockpit linux
+slug: cockpit-179
+category: release
+summary: Detach disk from VM with LibvirtDBus provider and offer cockpit-machines as Application
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 179.
+
+### Machines: Detach disk support
+
+The Machines page now supports detaching a disk from VMs with the LibvirtDBus
+provider.
```

Comment:
This is a bit misleading, as the provider doesn't apply to the VM itself. How about "When using the libvirt D-Bus provider, the  Machines page now supports detaching a disk from VMs."